app/views/customers/_testreserved_regular_voucher.html.haml

```diff
@@ -0,0 +1,17 @@
+- if @gAdminDisplay || group.cancelable_by(@customer) then
+    %td.c= menu_or_static_text('cancelnumber', group.size)
+    %td{:title => (group.voucherlist if @gAdminDisplay)}=   group.name
```

Comment:
Why is group thing here? One branch should do one thing?

Reply:
Nothing with group should be there. I don't know what that is but I didn't write that